File: app/src/main/java/com/example/marvel/adapter/ViewHolderPersonagens.java
package com.example.marvel.adapter;

import android.view.View;
import android.widget.ImageView;
import android.widget.TextView;
import androidx.annotation.NonNull;
import androidx.cardview.widget.CardView;
import androidx.recyclerview.widget.RecyclerView;

import com.example.marvel.R;


public class ViewHolderPersonagens extends RecyclerView.ViewHolder {

    ImageView imageView;
    TextView heroi;
    TextView nome;
    CardView cardViewCharacters;

    public ViewHolderPersonagens(@NonNull View itemView) {
        super(itemView);

        heroi = itemView.findViewById(R.id.text_titulo);
        nome = itemView.findViewById(R.id.text_volume);
        cardViewCharacters = itemView.findViewById(R.id.card_view_characters);
        imageView = itemView.findViewById(R.id.imagePersonagens);
    }
}

File: app/src/main/java/com/example/marvel/adapter/ViewHolderPersonagens.kt
package com.example.marvel.adapter

import android.view.View
import android.widget.ImageView
import android.widget.TextView
import androidx.cardview.widget.CardView
import androidx.recyclerview.widget.RecyclerView
import com.example.marvel.R

class ViewHolderPersonagem(itemView: View) : RecyclerView.ViewHolder(itemView) {
    var imageView: ImageView = itemView.findViewById(R.id.imageCharacters)
    var heroi: TextView = itemView.findViewById(R.id.text_characters)
    var cardViewCharacters: CardView = itemView.findViewById(R.id.card_view_characters)
}